spartan/scripts/consolidate-sepolia-balances.sh

```diff
@@ -26,6 +26,12 @@ if ! command -v cast &>/dev/null; then
     $XDG_CONFIG_HOME/.foundry/bin/foundryup && export PATH="$PATH:$XDG_CONFIG_HOME/.foundry/bin"
```

Comment:
why is this copy and pasted in two places, include it in a config map?

Reply:
I swear there was a reason but I can't remember now so will happily update 👍

Reply:
remember now, it can't be in a config map bc it runs `post-delete` and these resources are removed during helm uninstall and it can't be found. That's the error I was getting at least, if you know a fix for that will implement.

I also kept it as a standalone script bc it's useful to call manually